app/classifier/drawing-tools/freehand-shape.cjsx

```diff
@@ -0,0 +1,69 @@
+React = require 'react'
+ReactDOM = require 'react-dom'
```

Comment:
Any reason `ReactDOM` is included here? I noticed it isn't in the other component.